config/settings-production.yml

```diff
@@ -19,6 +19,8 @@ buffy:
       authorized_roles_in_issue:
         - author1
         - author-others
+      remove_labels:
+        - 4/review(s)-in-awaiting-changes
```

Comment:
no, this is for the author submit response responder

Reply:
But my comment still applies. Plus note line 16:
https://github.com/ropensci-org/buffy/blob/044d630b554a7963aa2a1d8febe9422edd5643bd/config/settings-production.yml#L16
So that badge is *added* by this responder, but only once all reviews are in. This responder should not remove it.